src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/OrganisationEntityResponse.java

```diff
@@ -49,10 +49,14 @@ private void getOrganisationEntityResponse(Organisation organisation, Boolean is
         if (!organisation.getUsers().isEmpty()) {
             this.superUser = new SuperUserResponse(organisation.getUsers().get(0));
         }
-        this.paymentAccount = organisation.getPaymentAccounts()
-                .stream()
-                .map(pbaAccount -> new PbaAccountResponse(pbaAccount).getPbaNumber())
-                .collect(toList());
+        if (!StringUtils.isEmpty(organisation.getPaymentAccounts()) && organisation.getPaymentAccounts().size() > 0) {

```

Comment:
as per sonar lint we should avoid ! negation. sonar suggest use Boolean.FALSE.equals

Reply:
! negation is not required here so removed.